index.html

```diff
@@ -4,6 +4,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>React App using Parceljs</title>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=Array.from,Promise,Symbol,Object.setPrototypeOf,Object.getOwnPropertySymbols"></script>
+    <script src="https://cdn.jsdelivr.net/npm/superagent"></script>
```

Comment:
Superagent pulled from CDN over `node_modules`
https://github.com/visionmedia/superagent#vanillajs
Why?
Superagent has two versions one for the browser, one running on a server. The browser version uses XHR, which MirageJS can intercept. The Node version is incompatible with Mirage. When running tests, superagent defaults to the Node version.

As a result we have to tell our tests to explicitly use the browser version
https://github.com/adriene-orange/react-sandbox/pull/10/files#diff-158e440b2525ccadb7d4bf2199698a9dR6